apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -159,10 +159,14 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
         ).filter((item: ProfileSchemaInterface) =>
             item.name !== ProfileConstants?.SCIM2_SCHEMA_DICTIONARY.get("META_VERSION")
         ).sort((a: ProfileSchema, b: ProfileSchema) => {
-            if (!a.displayOrder) {
-                return -1;
-            } else if (!b.displayOrder) {
+            if (!a.displayOrder) {  
+                if (a.displayName === "Username"){
```

Comment:
Better to move this to a constant in a follow up PR.
Also i'm not sure if this'll scale since the claim displayname can be customized.